test/browser/features/integrity.feature

```diff
@@ -0,0 +1,19 @@
+@skip_ie_11 @skip_ios_10 @skip_safari_10 @skip_safari_16 @skip_edge_17 @skip_chrome_43
+Feature: Bugsnag-Integrity header
+
+Scenario: Integrity headers are set when setPayloadChecksums is true
+  When I navigate to the test URL "/integrity/script/a.html"
```

Comment:
could we name the html something closer to what the test actually does? otherwise we have to reference between the feature and the fixture to figure out what `a.html` does

Reply:
yeah, that sounds good